include/boost/url/authority_view.hpp

```diff
@@ -199,11 +199,18 @@ class BOOST_SYMBOL_VISIBLE
         return BOOST_URL_MAX_SIZE;
     }
 
-    /** Return the number of characters in the authority.
+    /** Return the number of characters in the authority
 
-        This function returns the number
-        of characters in the authority, which
-        does not include a null terminator.
+        This function returns the number of
+        characters in the authority, not
+        including any null terminator, if present.
```

Comment:
the authority view cannot promise a null terminator

Reply:
It's actually promising _NOT_ to include the null terminator. It might have a null terminator because it can now be constructed directly from a string.